### DISCUSSION TITLE
2022 · 逆向到Web开发 - 愧怍的小站

### DISCUSSION THREAD

**Discussion by giscus** [kuizuo/blog#192] [General]
# 2022 · 逆向到Web开发 - 愧怍的小站

距离上一篇博文有一个月之久，距离上次编写代码也有一周之久，由于疫情封控全面放开，加上福建省教育厅通知的提前返乡，反而让我感到有些不适。往常这个时间点我忙于期末考试，会把代码的事情放一边，等到彻底放假后，开始闭关潜心学习。然而剩余的半个月变成了线上形式，课还是要上，考试还是要考，虽说身在家乡，但心不在焉的。

https://kuizuo.cn/2022-year-end-summary/#comments

**Comment by qingyi11** [comment]
干,你肯定报了桌斌的课;钱都花了,不继续干下去有点可惜了
"未来逆向的难度只增不减，但薪资可不一定保证这趋势。"如果真是这样我估计也要转开发了吧

**Reply by kuizuo** [reply]
我都差点成了小肩膀的优秀学员😂，不继续干下去主要还是太容易走偏了，至于说已经花费的时间与金钱倒没有什么可惜，能发挥它们所应有的价值才是关键。

**Reply by qingyi11** [reply]
还有优秀学员一说的吗

**Reply by kuizuo** [reply]
> 还有优秀学员一说的吗

优秀学员指头像变灰的学员，亦指吃lao饭的学员

**Comment by janzhou123** [comment]
可以去考研，现在热门方向很多，继续深造会必要好，前提是要选对方向，关键是自己感兴趣；
PS: 你这个评论貌似不能退出 ：（

**Reply by kuizuo** [reply]
说实话，国内的研如果不是名校的那种，读了就等于多混两年。而且如果考研的话，会有整整一段周期的时间要用于学习，而不是编写代码，这也是令我比较头疼的一点。

**Reply by janzhou123** [reply]
编写代码只是个工具，手段； 还需要往上去拓展，这些工具可以做的事情，这个才是最重要的；不同的学历就会有不同层次的视角和眼界，同时也会决定人的社会地位；决定所做事情的大小； 学习和写代码，不冲突；不要过多的宅男；

**Reply by Rehoni** [reply]
在国内的环境来说，学历是门槛，多少还是有点意义的。另外，混还是卷，这个或多或少取决于个人吧，像你这么优秀，可能不用担心太多混的问题。

**Comment by xiaowu55** [comment]
同大三下，你真的比我厉害很多很多。
‘我通常会用项目或者博文的形式来总结我的学习过程。我也乐意花时间在这上面，并将其分享出来。当有其他人也在学习这门技术时，看到该项目或博文，我就认为非常有意义。’因为是自学前端，像你这样的人帮助了我很多，万分感激。（因为想搭建一个博客找到的）

**Reply by kuizuo** [reply]
感谢你的夸奖，很高兴我的学习总结能够帮助到你。
如果你有任何问题或需要帮助，随时都可以联系我。共勉💪

**Reply by xiaowu55** [reply]
共勉~,祝身体健康hh

**Comment by yuantuo666** [comment]
docusaurus 的 showcase 来的，太强了 :+1:

**Comment by sanbuphy** [comment]
同福建,在用作者的模板魔改自己的小站中,可以加个微信认识下吗

**Reply by kuizuo** [reply]
微信号 kuizuo12，可以认识认识🤝
